samples/apps.json

```diff
@@ -11,6 +11,11 @@
             "webOriginUrl": "http://localhost:8000"
         }
     },
+    "affinidi-dotnet-openidconnect": {
+        "redirectUris": {
+            "callbackUrl": "http://localhost:5068/signin-oidc"
```

Comment:
Can we keep the callback url as /callback

Reply:
This I will take as next step as OIDC library uses this as default.